charts/incubator/hyperswitch-app/templates/consumer/configmap.yaml

```diff
@@ -3,490 +3,6 @@ kind: ConfigMap
-  ROUTER__REPORT_DOWNLOAD_CONFIG__REFUND_FUNCTION: "weekly_refund_report_generator_step_1"
-  ROUTER__REPORT_DOWNLOAD_CONFIG__REGION: "eu-central-1"
-  ROUTER__SCHEDULER__CONSUMER_GROUP: "SCHEDULER_GROUP"
-  ROUTER__SCHEDULER__CONSUMER__CONSUMER_GROUP: "SCHEDULER_GROUP"
-  ROUTER__SCHEDULER__CONSUMER__DISABLED: "false"
-  ROUTER__SCHEDULER__GRACEFUL_SHUTDOWN_INTERVAL: "60000"
-  ROUTER__SCHEDULER__LOOP_INTERVAL: "30000"
-  ROUTER__SCHEDULER__SERVER__HOST: "0.0.0.0"
-  ROUTER__SCHEDULER__SERVER__PORT: "3000"
-  ROUTER__SCHEDULER__SERVER__WORKERS: "1"
-  ROUTER__SCHEDULER__STREAM: "SCHEDULER_STREAM"
-  ROUTER__SECRETS_MANAGEMENT__SECRETS_MANAGER: {{ .Values.application.server.secrets_manager | quote }}
-  ROUTER__SERVER__BASE_URL:
-    {{ .Values.application.server.server_base_url | quote }}
-  ROUTER__SERVER__REQUEST_BODY_LIMIT: "32768"
-  ROUTER__SERVER__WORKERS: "8"
-  ROUTER__TEMP_LOCKER_ENABLE_CONFIG__BANKOFAMERICA__PAYMENT_METHOD: "card"
-  ROUTER__TEMP_LOCKER_ENABLE_CONFIG__BLUESNAP__PAYMENT_METHOD: "card"
-  ROUTER__TEMP_LOCKER_ENABLE_CONFIG__CYBERSOURCE__PAYMENT_METHOD: "card"
-  ROUTER__TEMP_LOCKER_ENABLE_CONFIG__NMI__PAYMENT_METHOD: "card"
-  ROUTER__TEMP_LOCKER_ENABLE_CONFIG__NUVEI__PAYMENT_METHOD: "card"
-  ROUTER__TEMP_LOCKER_ENABLE_CONFIG__PAYME__PAYMENT_METHOD: "card"
-  ROUTER__TEMP_LOCKER_ENABLE_CONFIG__SHIFT4__PAYMENT_METHOD: "card"
-  ROUTER__TEMP_LOCKER_ENABLE_CONFIG__STRIPE__PAYMENT_METHOD: "bank_transfer"
-  ROUTER__TOKENIZATION__BRAINTREE__LONG_LIVED_TOKEN: "false"
-  ROUTER__TOKENIZATION__BRAINTREE__PAYMENT_METHOD: "card"
-  ROUTER__TOKENIZATION__CHECKOUT__APPLE_PAY_PRE_DECRYPT_FLOW: "network_tokenization"
-  ROUTER__TOKENIZATION__CHECKOUT__LONG_LIVED_TOKEN: "false"
-  ROUTER__TOKENIZATION__CHECKOUT__PAYMENT_METHOD: "wallet"
-  ROUTER__TOKENIZATION__GOCARDLESS__LONG_LIVED_TOKEN: "true"
-  ROUTER__TOKENIZATION__GOCARDLESS__PAYMENT_METHOD: "bank_debit"
-  ROUTER__TOKENIZATION__MOLLIE__LONG_LIVED_TOKEN: "false"
-  ROUTER__TOKENIZATION__MOLLIE__PAYMENT_METHOD: "card"
-  ROUTER__TOKENIZATION__PAYME__LONG_LIVED_TOKEN: "false"
-  ROUTER__TOKENIZATION__PAYME__PAYMENT_METHOD: "card"
-  ROUTER__TOKENIZATION__SQUARE__LONG_LIVED_TOKEN: "false"
-  ROUTER__TOKENIZATION__SQUARE__PAYMENT_METHOD: "card"
-  ROUTER__TOKENIZATION__STAX__LONG_LIVED_TOKEN: "true"
-  ROUTER__TOKENIZATION__STAX__PAYMENT_METHOD: "card,bank_debit"
-  ROUTER__TOKENIZATION__STRIPE__LONG_LIVED_TOKEN: "false"
-  ROUTER__TOKENIZATION__STRIPE__PAYMENT_METHOD: "wallet"
-  ROUTER__TOKENIZATION__STRIPE__PAYMENT_METHOD_TYPE__LIST: "google_pay"
-  ROUTER__TOKENIZATION__STRIPE__PAYMENT_METHOD_TYPE__TYPE: "disable_only"
-  ROUTER__WEBHOOKS__OUTGOING_ENABLED: "true"
-  ROUTER__WEBHOOK_SOURCE_VERIFICATION_CALL__CONNECTORS_WITH_WEBHOOK_SOURCE_VERIFICATION_CALL: "paypal"
-  RUN_ENV: {{ .Values.application.server.run_env | quote }}
-  RUST_MIN_STACK: "4194304"
-  SCHEDULER_FLOW: "consumer"
+  sandbox.toml: |
```

Comment:
Let us name the configmaps with a generic name instead of using environment names like sandbox/production.